jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/resources/SessionManagerPluginInstallationVerfication.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.ecs.resources
+
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.process.CapturingProcessHandler
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
+
+object SessionManagerPluginInstallationVerfication {
+    fun checkInstallation(): Int = runBlocking(Dispatchers.IO) {
```

Comment:
I would make this return boolean instead of the exit code, the exit code is an implementation detail of if the installation is valid

Reply:
Makes sense, changed it!